src/main/menus/app.js

```diff
@@ -136,6 +136,13 @@ var createTemplate = function(mainWindow, config) {
       click: () => {
         mainWindow.webContents.send('zoom-in', -1);
       }
+    }, {
+      label: 'Zoom Out (hidden)',
+      accelerator: 'CmdOrCtrl+_',
+      visible: false,
+      click: () => {
```

Comment:
@kytwb #240 can use `role` with custom label and accelerator. Would you try? Roles are listed at http://electron.atom.io/docs/api/menu-item/ .